quickstarts/blazor-web-assembly/config.yml

```diff
@@ -0,0 +1,53 @@
+
+
+slug: blazor-web-assembly
+description: |
+  ## Comprehensive monitoring quickstart for Blazor Web Assembly
+
+  Microsoft developed Blazor Web Assembly to build web UIs with C#, HTML, and CSS. As a feature of ASP.NET, our Blazor Web Assembly integration with the Browser Monitoring agent lets you monitor your apps with our quickstart dashboards right out of the box.
```

Comment:
Blazor WebAssembly

Reply:
There's this, to me, frustrating thing that these quickstart pages do. I don't think we need to tell someone what Blazor WebAssembly is to people on the New Relic I/O page. They're here to learn about what New Relic can do for them!

How about something more like: 

"As you probably know, Microsoft developed Blazor WebAssembly to build web UI via C#, HTML, and CSS. With our browser monitoring agent, you can monitor your Blazor WebAssembly app performance with our pre-built dashboards." or something.